### ISSUE TITLE
Mayan Long Count Bonus Suggestion

### ISSUE THREAD

**Issue by Hawawaa** [Thalassicus/cep-bnw#35]
When you get your choice for a free great person you can only click ok. I would be good if you could add a cancel button to double check what great person you need.

**Comment by Thalassicus** [comment]
I agree this would be nice. I'm rather unfamiliar with UI code, so I don't know how difficult this would be.
